### DISCUSSION TITLE
grand differences of LTC for different supercells

### DISCUSSION THREAD

**Discussion by Syaooooo** [ttadano/alamode#52] [Q&A]
Dear Prof.Tadano,

sorry to bother, recently I used 'ALAMODE' to calculate lattice thermal conductivity (LTC) and met some unclear points want to discuss with you,
the structure is LiBeP with Pnma phase (orthorhombic structure), after relaxation the POSCAR like this:
(CONTCAR -> BPOSCAR)
====
generated by phonopy
   1.0
     6.2730828425137952    0.0000000000000000    0.0000000000000000
     0.0000000000000000    3.6495032251349779    0.0000000000000000
     0.0000000000000000    0.0000000000000000    6.5143414059868077
Li Be P
   4    4    4
Direct
  0.0153079530240116  0.2500000000000000  0.1752023755985793
 ..
====
consider it's time-consuming for calculating LTC, I tried the supercell for 2x2x2, but it failed to work,
then I tried 2x1x2, 1x2x1, 2x2x1 these 3 supercell,
but surprisedly the 3 results shows big differences along the direction xx, yy, zz:
cause for the q points with 30 and 40 still hard to finish I just put 10 and 20 here:
-----
                  2x1x2                        1x2x1                   2x2x1
q10    13.14/25.80/17.17      8.18/15.37/12.59      7.21/13.83/10.31
q20    14.12/26.97/19.51     8.66/16.02/13.05     7.63/14.50/10.90
-----
Theoretically the LTC is an intrinsic properties for a compound, it shouldn't have such grand differences between the supercells, what makes me deeply confused.

Attached I put my .in document (under 2x1x2 supercell, the other sets are similar).

I'm appreciate with your patience and help,
and look forward your reply.
Lin

[RTA-in.txt](https://github.com/ttadano/alamode/files/7595750/RTA-in.txt)
[scph-in.txt](https://github.com/ttadano/alamode/files/7595751/scph-in.txt)
[opt-in.txt](https://github.com/ttadano/alamode/files/7595752/opt-in.txt)
[CV-in.txt](https://github.com/ttadano/alamode/files/7595753/CV-in.txt)

**Comment by ttadano** [comment]
Hi,

Have you compared the phonon dispersion curves obtained with these three settings? I guess there are significant differences in the group velocities as well.

**Reply by Syaooooo** [reply]
Dear Prof.Tadano,
Here are the comparison for harmonic phonon dispersion / self-consistent phonon dispersion (300K) / group velocities under 300K, 
also while painting phonon dispersion diagram I used same primitive cell, 

what makes me confused is theoretically that could not have such grand differences, when I used same  parameters and the only change is the size of supercell,
is there any set from my side become wrong or any other original set of ALAMODE I ignored?

I'm appreciate for your reply and patience,
Lin

![SCPH-300K](https://user-images.githubusercontent.com/87617566/143420272-337d8eb0-6fc1-43f2-9b9d-1275b5a77d48.jpg)
![Harmonic phonon](https://user-images.githubusercontent.com/87617566/143420296-b696bc85-53e0-49d3-bdac-ecad04db52f1.jpg)
![q10-group velocity](https://user-images.githubusercontent.com/87617566/143420299-83952aa0-2e80-4579-a46e-6e99ed259370.jpg)

**Reply by ttadano** [reply]
Thank you for the figures. 

Perhaps, the 2x1x2 supercell was too short along the b-axis, which could result in underestimation of the ph-ph scattering rates. The 1x2x1 and 2x2x1 results are reasonably similar to each other, so I think the 2x1x2 result is exceptional. 

I would suggest using the 1x2x1 supercell for computing the 3rd-order force constants and the 2x2x2 supercell for the 2nd-order. 
By using the FC2XML tag in ANPHON, you can combine harmonic and anharmonic force constants computed with different supercells.

**Reply by Syaooooo** [reply]
Dear Prof.Tadano,

I'm appreciate for your nice suggestion! That gives me very sparkly inspiration,
I finished the calculation for q10, it seems similar with previous results,
the issue is when I try to combine the FCs, there are lots of data cannot connect, but at least it works.

Shuyao
<img width="523" alt="after dfc2 to build high-order FC" src="https://user-images.githubusercontent.com/87617566/144009308-bd6cf8e1-57df-4d75-9807-f4b33dbbc788.png">
<img width="246" alt="Results" src="https://user-images.githubusercontent.com/87617566/144009356-0a5e4203-3fed-4f64-8b01-12192f379733.png">
